app.js

```diff
@@ -8,8 +12,26 @@ app.get("/api/topics", getTopics);
 
 app.get("/api", getEndpoints);
 
+app.get("/api/articles/:articles_id", getArticlesById);
+
 app.all("*", (request, response) => {
   response.status(404).send({ msg: "Path Not Found" });
 });
 
+app.use((err, req, res, next) => {
```

Comment:
You may wish to extract your error handling into its own file shortly to encapsulate some of that logic :smile: